meinberlin/apps/budgeting/assets/BudgetingProposalList.jsx

```diff
@@ -4,23 +4,23 @@ import { BudgetingProposalListItem } from './BudgetingProposalListItem'
 import { Pagination } from './Pagination'
 import { CountDown } from '../../contrib/assets/CountDown'
 import { ControlBar } from './ControlBar'
-
-const nothingStr = django.gettext('Nothing to show')
+import { useLocation, useSearchParams } from 'react-router-dom'
 
 export const BudgetingProposalList = (props) => {
   const [data, setData] = useState([])
   const [meta, setMeta] = useState()
-  const [queryString, setQueryString] = useState('')
+  const location = useLocation()
+  const [queryParams] = useSearchParams()
 
-  const fetchProposals = (newIndex) => {
-    const pageNumber = newIndex || 1
-    const url = `${props.proposals_api_url}?page=${pageNumber}${queryString}`
+  const fetchProposals = () => {
+    const url = `${props.proposals_api_url}${location.search}`
```

Comment:
personal pref: as there is no need for the template here just use `props.proposals_api_url + location.search` ?